src/promptflow/MANIFEST.in

```diff
@@ -1,5 +1,6 @@
 include promptflow/azure/resources/*
 include promptflow/_sdk/_serving/static/*
+include promptflow/_sdk/_service/static/*
```

Comment:
dulplicate line?

Reply:
no, L2 refers to serving, while L3 is for service - you should have run into some issues with the package built 2 hours ago...